test/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ function(build_umf_test)
         set(INC_DIRS ${INC_DIRS} ${LEVEL_ZERO_INCLUDE_DIRS})
     endif()
 
```

Comment:
perhaps we should try and catch this in CI - e.g. in MacOS-14 job we could explicitly remove hwloc from the system (it's installed by default, apparently)

Reply:
Let's check: https://github.com/ldorau/unified-memory-framework/actions/runs/13074059175

Reply:
@lukaszstolarczuk it does not catch that case

Reply:
hmm that's weird... but let's call it a CI issue

Your fix is good, so LGTM